test/test_xcpp_kernel.py

```diff
@@ -132,6 +133,43 @@ class XCppTests(jupyter_kernel_test.KernelTests):
             }
         ]
 
+    #Helper function to execute the notebooks
+    def execute_notebook(inp, out):
+
+        # Execute the notebook
+        executed_notebook = pm.execute_notebook(
+            inp,
+            out,
+            log_output=True,
+            kernel_name='xcpp20'
+        )
+
+        # Read the input and output notebooks
+        with open(inp) as f:
+            input_nb = nbformat.read(f, as_version=4)
+        with open(out) as f:
+            output_nb = nbformat.read(f, as_version=4)
+
+        check = True
+
+        # Iterate over the cells in the input and output notebooks
+        for input_cell, output_cell in zip(input_nb.cells, output_nb.cells):
+            if input_cell.cell_type == 'code' and output_cell.cell_type == 'code':
+                # Check the outputs of the cells
+                for input_output, output_output in zip(input_cell.outputs, output_cell.outputs):
+                    if input_output.get('text') != output_output.get('text'):
+                        check = False
+                        break
+        
+        return check
+
+    # Tests for Notebooks
+    class XCppNotebookTests(unittest.TestCase):
+
+        def test_simple_notebook(self):
+            self.assertEqual(execute_notebook('Notebooks/Untitled.ipynb', 'Notebooks/Untitled_output.ipynb'), True)
```

Comment:
What happens if you change the input expected values, will this break?

Reply:
Yes.

Reply:
Can you make that change and see if the CI fails?

Reply:
Ping.

Reply:
I still would like to introduce a different expected result in the test notebook and see the CI of this PR fail first.